docs/workflow/implement-custom-activity-server.mdx

```diff
@@ -46,7 +46,7 @@ namespace App1.Workflow.CustomActivity
 ```
 
 :::tip
-Use a unique prefix on the `Action` property to avoid it conflicting with someone else's custom activities.  
+Use a unique prefix on the `Action` property to avoid it conflicting with someone elses custom activities.  
```

Comment:
I don't think this change was correct.

Reply:
It wasn't.... the spellchecker was mistaken, it told me it was.
reverted and added an exception to the spellchecker.